src/cmd/base.js

```diff
@@ -0,0 +1,35 @@
+const { errors: { usageError } } = require('../app/command-processor');
+const spinnerMixin = require('../lib/spinner-mixin');
+const UI = require('../lib/ui');
+
+const DEVICE_ID_PTN = /^[0-9a-f]{24}$/i;
```

Comment:
Should we be more permissive and accept capital hex letter too? The API lowercases it before doing the database query.

Reply:
>Should we be more permissive and accept capital hex letter too?

it accepts capitals (note the `/i` modifier) - i added an assertion to make that clearer 👉 https://github.com/particle-iot/particle-cli/pull/570/commits/1c3b12df6e1e9f725f73b581c63edfd9e7cdf573#diff-d3d434b0fb05ede651109216b9967a98R31